Framework/Algorithms/src/Stitch1DMany.cpp

```diff
@@ -68,7 +74,9 @@ void Stitch1DMany::init() {
   declareProperty(make_unique<PropertyWithValue<int>>(
                       "ScaleFactorFromPeriod", 1,
                       scaleFactorFromPeriodValidator, Direction::Input),
-                  "Provided index of period to obtain scale factor from.");
+                  "Provided index of period to obtain scale factor from. "
+                  "Periods are indexed from 1. Used only if stitching group "
+                  "workspaces and UseManualScaleFactor is false.");
```

Comment:
I am not sure that `ScaleFactorFromPeriod` is used if `UseManualScaleFactor` is `False`, I think it is the other way around: `UseManualScaleFactor` has to be `True` in order for the algorithm to use `ScaleFactorFromPeriod`.

The following script illustrates this:

```
import numpy as np

def straight_line(x, a, b):
  return a+b*x

#create two histograms
x1 = np.arange(-1, 1, 0.02)
x2 = np.arange(0.4, 1.6, 0.02)
x3 = np.arange(1.3, 3, 0.02)
ws_1_1 = CreateWorkspace(UnitX="1/q", DataX=x1, DataY=straight_line(x1[:-1], 1, 1)+1)
ws_2_1 = CreateWorkspace(UnitX="1/q", DataX=x2, DataY=straight_line(x2[:-1], 1, 1)+1)
ws_3_1 = CreateWorkspace(UnitX="1/q", DataX=x3, DataY=straight_line(x3[:-1], 1, 1)+1)
ws_1_2 = CreateWorkspace(UnitX="1/q", DataX=x1, DataY=straight_line(x1[:-1], 1, 1)+1)
ws_2_2 = CreateWorkspace(UnitX="1/q", DataX=x2, DataY=straight_line(x2[:-1], 1.5, 1)+1)
ws_3_2 = CreateWorkspace(UnitX="1/q", DataX=x3, DataY=straight_line(x3[:-1], 2, 1)+1)
ws_1_3 = CreateWorkspace(UnitX="1/q", DataX=x1, DataY=straight_line(x1[:-1], 1, 1)+1)
ws_2_3 = CreateWorkspace(UnitX="1/q", DataX=x2, DataY=straight_line(x2[:-1], 2, 1)+1)
ws_3_3 = CreateWorkspace(UnitX="1/q", DataX=x3, DataY=straight_line(x3[:-1], 3, 1)+1)
group1 = GroupWorkspaces('ws_1_1, ws_1_2, ws_1_3')
group2 = GroupWorkspaces('ws_2_1, ws_2_2, ws_2_3')
group3 = GroupWorkspaces('ws_3_1, ws_3_2, ws_3_3')

stitched_from_period, factors = Stitch1DMany('group1, group2, group3', Params=[0.02], UseManualScaleFactor=True, ScaleFactorFromPeriod=1)
print factors
stitched_from_period, factors = Stitch1DMany('group1, group2, group3', Params=[0.02], UseManualScaleFactor=False, ScaleFactorFromPeriod=1)
print factors
```

In the first `Stitch1DMany` I set `UseManualScaleFactor=True` and the rest of the periods are stitched with scale factor `1` (because data in the first period of each group is the same, so their scale factor is `1`). In the next line, `UseManualScaleFactor=True` and the output scale factors are different for the different periods.